experiment/measurer.py

```diff
@@ -432,6 +547,25 @@ def generate_summary(self, cycle: int):
                 'Coverage summary json file generation failed for \
                     cycle: %d.', cycle)
 
+    def generate_coverage_information(self, cycle: int):
+        """Generate the .profdata file and then transform it into
+        json summary."""
+        if not os.path.exists(self.profraw_file):
```

Comment:
Didn't abhishek make this change? WHy is it in this PR?

Reply:
It is to solve [#589 (comment)](https://github.com/google/fuzzbench/pull/589#discussion_r460548454) so that we won't have two duplicate error messages if profraw file is empty or doesn't exist.